### PR TITLE
i18n: Fix translatable strings as discovered by our translators (Week 12)

### PR DESCRIPTION
**Pull request by akirk**
These strings are hard to translate, this improves them.

### CODE REVIEW

File: client/components/tinymce/plugins/contact-form/dialog/settings.jsx [hunk 1]
```diff
@@ -42,14 +42,11 @@ export default React.createClass( {
...
-								args: {
-									postType: this.props.postType === 'post' ? this.translate( 'post' ) : this.translate( 'page' )
-								}
-							}
-						) }
+					{
+						this.props.postType === 'post'
+						? this.translate( 'If you don’t make any changes here, feedback will be sent to the author of the post and the subject will be the name of this post.' )
+						: this.translate( 'If you don’t make any changes here, feedback will be sent to the author of the page and the subject will be the name of this page.' )
+					}
```
Comment (akirk) [inline client/components/tinymce/plugins/contact-form/dialog/settings.jsx]: Explanation: For example in German "page" ("Seite") is feminine and "post" masculine ("Beitrag"). 

- `this.translate( 'post' )` would be `Beitrag`
- `this.translate( 'page' )` would be `Seite`
- `this.translate( '...author of the post...' )` would be `Autor des Beitrags`
- `this.translate( '...author of the page...' )` would be `Autor der Seite`
- `this.translate( '...author of the %(postType)s' )` needs to be `Autor des %(postType)s` or  `Autor der %(postType)s` which is impossible

**Comment by akirk** [comment]
FYI
@seear client/my-sites/themes/sheet.jsx
@ockham client/my-sites/themes/controller/index.node.js
@rodrigoi client/components/tinymce/plugins/contact-form/dialog/field-header.jsx and client/components/tinymce/plugins/contact-form/dialog/settings.jsx
@drewblaisdell client/my-sites/upgrades/checkout-thank-you/domain-registration-details.jsx

File: client/my-sites/themes/controller/index.node.js [hunk 2]
```diff
@@ -69,9 +69,8 @@ export function details( context, next ) {
 	const { slug } = context.params;
 	const user = getCurrentUser( context.store.getState() );
 	const themeName = ( getThemeDetails( context.store.getState(), slug ) || false ).name;
-	const title = i18n.translate( '%(theme)s Theme', {
-		args: { theme: themeName },
-		textOnly: true
```
Comment (akirk) [inline client/my-sites/themes/controller/index.node.js]: See https://github.com/Automattic/wp-calypso/pull/2461

**Comment by rodrigoi** [comment]
I can confirm that the changes on both `client/components/tinymce/plugins/contact-form/dialog/field-header.jsx` and `client/components/tinymce/plugins/contact-form/dialog/settings.jsx` look good. :shipit:

File: client/my-sites/themes/controller/index.node.js [hunk 3]
```diff
@@ -69,9 +69,8 @@ export function details( context, next ) {
...
+	const title = i18n.translate( '%(themeName)s Theme', {
+		args: { themeName: themeName }
```
Comment (ockham) [inline client/my-sites/themes/controller/index.node.js]: You could shorten that to `args: { themeName }`, see https://github.com/lukehoban/es6features#enhanced-object-literals.

Comment (akirk) [inline client/my-sites/themes/controller/index.node.js]: I always forget about that, thanks!

**Comment by seear** [comment]
Themes stuff looks good, thanks @akirk :+1:

**Comment by akirk** [comment]
@drewblaisdell could I get an OK from you, too? Thanks!

**Comment by drewblaisdell** [comment]
The comment looks good. :+1: Thanks for updating.

**Comment by lancewillett** [comment]
Hi @akirk — good to merge? 6 days old now.

**Pull request merged**